Week-2/1.Flight-booking.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+function getFullName(firstname, surname, useFormalName, gender) {
+  firstname = firstname.trim();
+  surname = surname.trim();
+  const fullName = `${firstname} ${surname}`;
+
+  if (!firstname || !surname) {
```

Comment:
Do your invalid checking first, even before assembling `fullName` ; it's fine here but in more complicated functions, not catching the invalid agad is going to break the assembling of more important things

Reply:
yeah I was actually debating this to myself kanina lols

Reply:
nice